projects/downloads.py

```diff
@@ -0,0 +1,158 @@
+from blueprintapi.oscal.oscal import Metadata
+from blueprintapi.oscal.ssp import (
+    BackMatter,
+    ByComponent,
+    Component,
+    ControlImplementation,
+    Impact,
+    ImplementedRequirement,
+    ImportProfile,
+    InformationType,
+    Model,
+    NetworkDiagram,
+    Party,
+    Resource,
+    Role,
+    SecurityImpactLevel,
+    SystemCharacteristics,
+    SystemImplementation,
+    SystemInformation,
+    SystemSecurityPlan,
+    SystemStatus,
+)
+from components.componentio import ComponentTools
+from projects.models import Project
+
+
+class OscalSSP: # pylint: disable=too-many-instance-attributes
```

Comment:
Instead of disabling the warning, we should think about if there's a better approach. For example, there was a different class that was reduced to a single function in a previous PR: https://github.com/CMSgov/blueprint_api/pull/85/ (`EmptyComponent` -> `create_empty_component_json`). 

Not necessarily saying that it needs to be a function per se, but it does look like the main functionality/usage for this class will be `get_ssp`

Reply:
In fact, I wonder if this would be better as a method on the `Project` model since it's effectively a transformation of a project.